subsys/bluetooth/controller/Kconfig

```diff
@@ -128,6 +128,15 @@ config BT_CTLR_TX_BUFFERS
 	  packets can be enqueued, with 18 packets with L2CAP payload size of 1
 	  byte can be acknowledged.
 
+config BT_CTLR_TX_CTRL_BUFFERS
```

Comment:
Lets calculate the worst possible in the controller, and not have this user configurable. Ideally, implementing quotas for control PDU used for local initiation and those used for response to remote initiated. And this per active connection.

Reply:
If we can come up with a one-size-fits all calculation I am fine with that. However, as far as I understand, different LLLs may behave differently with regard to how many buffers may be in flight before they are returned to the pool, which would make it impossible to find a number that makes everybody happy.